doc/source/API/ClientServer.rst

```diff
@@ -120,3 +120,36 @@ As an alternative, the user can upload the script to run to the server and run i
     upload(local_script, remote_script, "servername")
     ansysem = "/path/to/AnsysEMxxx/Linux64"
     my_client.root.run_script(remote_script, ansysem_path=ansysem)
+
+
+CPython on Linux with client-server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pyaedt can also be launched on a linux machine, locally to bypass current Ironpython limits. To do this,
+the steps are:
+
+1. Install pyaedt on linux using pip  (minimum version 0.4.23)
+2. Launche CPython and run pyaedt
+
+.. code:: python
+
+    # Launch the latest installed version of AEDT in graphical mode.
+
+    from pyaedt.common_rpc import launch_ironpython_server
+    client = launch_ironpython_server(ansysem_path="/path/to/ansys/executable/folder", non_graphical=True, port=18000)
```

Comment:
@maxcapodi78 My suggestion is just to be consistent with other `ansysem_path` but it might not be applicable here. Up to you.
```suggestion
    client = launch_ironpython_server(ansysem_path="/path/to/AnsysEMxxx/Linux64", non_graphical=True, port=18000)
```